libs/geography/src/facades/geography/geography.facade.spec.ts

```diff
@@ -17,8 +17,8 @@ import { DaffCountryFactory, DaffSubdivisionFactory } from '@daffodil/geography/
 import { DaffGeographyFacade } from './geography.facade';
 
 describe('DaffGeographyFacade', () => {
```

Comment:
These tests should continue to use jasmine marbles instead of subscribe just to avoid possible subscription memory leaks. You wouldn't think it would happen but I've found testing with rxjs and angular to be a little unpredictable in terms of what carries over between tests.

Reply:
I was using it so I could take advantage of different matchers like `toBeFalsy()` but with those new changes you requested, that shouldn't be necessary.

Reply:
changed